broker/src/test/java/org/apache/rocketmq/broker/filter/MessageStoreWithFilterTest.java

```diff
@@ -76,6 +86,24 @@
         }
     }
 
+    @Before
+    public void init() {
+        filterManager = ConsumerFilterManagerTest.gen(topicCount, msgPerTopic);
+        try {
+            master = gen(filterManager);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
redundant code, when you use assert in your exception scenario, right ?

Reply:
This is a DefaultMessageStore start progress that used in annotation Before.And can't add expect Exception in before,I think there is no need to add exception expect in before and all we want is while the messageStore is started or not.